eras/shelley/test-suite/test/Test/Cardano/Ledger/Shelley/Examples/TwoPools.hs

```diff
@@ -482,13 +480,13 @@ blockEx5 =
     (mkOCert Cast.alicePoolKeys 0 (KESPeriod 10))
 
 activeStakeEx5 :: Integer
-activeStakeEx5 = sum $ unCoin <$> [aliceCoinEx1, bobInitCoin, carlInitCoin]
+activeStakeEx5 = sum $ map unCoin [aliceCoinEx1, bobInitCoin, carlInitCoin]
```

Comment:
just curious, is this just stylistic, making it read easier?

Reply:
This is a very good question. It is actually not due to a stylistic choice, but rather because I turned on `OverloadedLists` extension and using `map` here instead of `fmap` will restrict its type to `[a]`, rather than `IsList (t a) => t a`. 

So instead of manually restricting the type:

```haskell
activeStakeEx5 = sum $ unCoin <$> ([aliceCoinEx1, bobInitCoin, carlInitCoin] :: [Coin])
```

I chose to use `map` that will do that for me :)

Reply:
ah, thanks for the explanation!